src/core/utils/qfieldcloudutils.cpp

```diff
@@ -63,12 +67,14 @@ const QString QFieldCloudUtils::getProjectId( const QString &fileName )
 
 void QFieldCloudUtils::setProjectSetting( const QString &projectId, const QString &setting, const QVariant &value )
 {
+  thread_local QgsSettings settings;
   const QString projectPrefix = QStringLiteral( "QFieldCloud/projects/%1" ).arg( projectId );
-  return QSettings().setValue( QStringLiteral( "%1/%2" ).arg( projectPrefix, setting ), value );
+  settings.setValue( QStringLiteral( "%1/%2" ).arg( projectPrefix, setting ), value );
```

Comment:
@suricactus did you have a reason to use QSettings() instead of QgsSettings()?

Reply:
No particular reason, I guess I was unaware at the time of writing.